src/db2/db2oltp.tcl

```diff
@@ -2306,3 +2306,57 @@ switch $myposition {
         }
     }
 }
+
+proc delete_db2tpcc {} {
+    global maxvuser suppo ntimes threadscreated _ED
+    upvar #0 dbdict dbdict
+    if {[dict exists $dbdict db2 library ]} {
+        set library [ dict get $dbdict db2 library ]
+    } else { set library "db2tcl" }
+    upvar #0 configdb2 configdb2
+    #set variables to values in dict
+    setlocaltpccvars $configdb2
+    if {[ tk_messageBox -title "Delete Schema" -icon question -message "Do you want to delete the [ string toupper $db2_dbase ] Db2 TPROC-C schema\nunder user [ string toupper $db2_user ]?" -type yesno ] == yes} { 
+        set maxvuser 1
+        set suppo 1
+        set ntimes 1
+        ed_edit_clear
+        set _ED(packagekeyname) "Db2 TPROC-C deletion"
+        if { [catch {load_virtual} message]} {
+            puts "Failed to created thread for schema deletion: $message"
+            return
+        }
+        .ed_mainFrame.mainwin.textFrame.left.text fastinsert end "#!/usr/local/bin/tclsh8.6
+#LOAD LIBRARIES AND MODULES
+set library $library
+"
+        .ed_mainFrame.mainwin.textFrame.left.text fastinsert end {if [catch {package require $library} message] { error "Failed to load $library - $message" }
+if [catch {::tcl::tm::path add modules} ] { error "Failed to find modules directory" }
+if [catch {package require tpcccommon} ] { error "Failed to load tpcc common functions" } else { namespace import tpcccommon::* }
+
+proc drop_schema { dbname user password } {
+    global tcl_platform
+        set force "db2 force applications all"
+        set drop "db2 drop database $dbname"
+    if {$tcl_platform(platform) == "windows"} {
+        set prefix "db2cmd -c -w -i"
```

Comment:
Same issue here.

Reply:
This has been tested and updated to just use the -c flag